add composition section including a chair (#467)
astropy/astropy-project: finance/process/policies.rst

   (Agreed `2021-09-17`__.)

   __ https://docs.google.com/document/d/1OpSEtJC0jQINTB-YNexxgnHX7-J6HRSkiPKYWBSCOfg/edit#bookmark=id.gkt3aebktlyb

> 
> 
> git@github.com:astropy/astropy-project.git
> 

Reviewer: I don't understand why git is mentioned here.
```suggestion
```

Reviewer 2: By mistake.

Composition of the Committee and Internal Duties
================================================

> #. The Finance Committee is composed of 4 people from the Astropy Community and an administrative support person as a non-voting member. The Committee may choose to have a Chair. Decisions that require votes of the Finance Committee (described in APE 19) may be made by three committee members; or two members, one of whom is the Chair. In the event of a tie (2 for and 2 against), the Chair’s vote breaks the tie. In most circumstances, the Chair should be the primary point of contact with NumFocus staff while keeping the entire committee aware of any communications. If none of the Committee members are members of the CoCo, the Chair typically will serve as the liaison to the CoCo.

Reviewer: Should we add the admin person to Team page? I only see 4 names listed but here you say admin person is also part of the committee.

```suggestion
#. The Finance Committee is composed of 4 people from the Astropy Community and an administrative support person as a non-voting member. The Committee may choose to have a Chair. Decisions that require votes of the Finance Committee (described in `APE 19 <https://github.com/astropy/astropy-APEs/blob/main/APE19.rst>`_) may be made by three committee members; or two members, one of whom is the Chair. In the event of a tie (2 for and 2 against), the Chair’s vote breaks the tie. In most circumstances, the Chair should be the primary point of contact with NumFOCUS staff while keeping the entire committee aware of any communications. If none of the Committee members are members of the CoCo, the Chair typically will serve as the liaison to the CoCo.
```

   (Agreed `2021-09-17`__.)

   __ https://docs.google.com/document/d/1OpSEtJC0jQINTB-YNexxgnHX7-J6HRSkiPKYWBSCOfg/edit#bookmark=id.gkt3aebktlyb

Composition of the Committee and Internal Duties
================================================

> #. The Finance Committee is composed of 4 people from the Astropy Community and an administrative support person as a non-voting member. The Committee may choose to have a Chair. Decisions that require votes of the Finance Committee (described in `APE 19 <https://github.com/astropy/astropy-APEs/blob/main/APE19.rst>`_) may be made by three committee members; or two members, one of whom is the Chair. In the event of a tie (2 for and 2 against), the Chair's vote breaks the tie. In most circumstances, the Chair should be the primary point of contact with NumFOCUS staff while keeping the entire committee aware of any communications. If none of the Committee members are members of the CoCo, the Chair typically will serve as the liaison to the CoCo.

Reviewer 3: Thanks for the suggestion! I don't feel strongly, just looking for the most efficient workflow. How about: in the event of a tie, we defer to the CoCo?

> #. The Finance Committee is composed of 4 people from the Astropy Community and an administrative support person as a non-voting member. The Committee may choose to have a Chair. Decisions that require votes of the Finance Committee (described in `APE 19 <https://github.com/astropy/astropy-APEs/blob/main/APE19.rst>`_) may be made by three committee members; or two members, one of whom is the Chair. In the event of a tie (2 for and 2 against), the Chair's vote breaks the tie. In most circumstances, the Chair should be the primary point of contact with NumFOCUS staff while keeping the entire committee aware of any communications. If none of the Committee members are members of the CoCo, the Chair typically will serve as the liaison to the CoCo.

Author: ```suggestion
#. The Finance Committee is composed of 4 people from the Astropy Community and an administrative support person as a non-voting member. The Committee may choose to have a Chair. Decisions that require votes of the Finance Committee (described in `APE 19 <https://github.com/astropy/astropy-APEs/blob/main/APE19.rst>`_) may be made by three committee members; or two members, one of whom is the Chair. In the event of a tie (2 for and 2 against), the committee can defer to the CoCo for a decision or let the item fail. In most circumstances, the Chair should be the primary point of contact with NumFOCUS staff while keeping the entire committee aware of any communications. If none of the Committee members are members of the CoCo, the Chair typically will serve as the liaison to the CoCo.
```
